packages/cli/build/dev.ts

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ts-node
+import { spawnSync } from 'child_process'
+import onMain from './main'
+import { build, dev } from './typescript'
+
+async function main() {
+    await build().catch(console.error)
+    dev()
+    if (process.argv[2] === '--') {
+        spawnSync(process.argv.slice(3).join(' '), { shell: true, stdio: 'inherit' })
+    } else {
+        onMain('dev')
```

Comment:
Suggestion: If the `onMain` or `spwanSync` dies, quit the process with that error